aws/ecs/README.md

```diff
@@ -1,75 +1,49 @@
 
-- `dockerRegistryAuthToken` - Authentication token to `docker.cke-cs.com` repository
-- `version` - Collaboration Server version (default `latest`)
-- `env` - A comma-separated list of CKEditor Collaboration Server configuration ([see available options](https://ckeditor.com/docs/cs/latest/onpremises/cs-onpremises/installation/docker.html#docker-container-environment-variables))
+- Network resources: VPC, subnets, route tables, internet and NAT gateways.
+- Load balancer and ECS cluster with ECS task running CKEditor Collaboration Server On-Premises application.
+- Redis and MySQL databases in private subnets.
+- S3 bucket for Easy Image and collaboration storage.
 
-## Quick start
+## Usage
+Ensure you have terraform in version `1.7.4` or higher and your AWS credentials are configured properly.
 
-To use AWS CDK first You need an AWS account and AWS credentials. Read more about how to configure the environment [on official CDK docs](https://docs.aws.amazon.com/cdk/latest/guide/work-with.html).
-
-- install dependencies
-```bash
-	npm install
+1. Clone the repository
 ```
-
-- [bootstrap CDK](https://docs.aws.amazon.com/cdk/latest/guide/bootstrapping.html) into your AWS account
-```bash
-	npm run -- cdk bootstrap
+git clone git@github.com:cksource/ckeditor-cs-on-premises-infrastructure.git
 ```
 
-- deploy the application (replace `xxx` with your credentials).
-```bash
-	npm run -- cdk deploy -c dockerRegistryAuthToken=xxx -c env=LICENSE_KEY=xxx,ENVIRONMENTS_MANAGEMENT_SECRET_KEY=xxx
+2. Initialize terraform
+```
+cd aws/ecs
+terraform init
 ```
 
-**Note**:
-For easier work with CDK, we recommend using it together with AWS credentials manager like [aws-vault](https://github.com/99designs/aws-vault).
-## Infrastructure overview
-
-![CKEditor ECS Diagram](diagram.jpg)
-
-To provide high availability and fault tolerance we're recommending running the application in at least two availability zones. For high security, we're following the least privilege model for all of the infrastructure resources - no permissions are given unless they are required. All of the data used by the system is encrypted.
-
-### Load balancer
-Spread traffic evenly between servers. This is the only public (internet facing) component.
-
-### ECS
-Docker container orchestration for running the application and managing available EC2 resources.
-
-### Database
-Databases are running in an isolated network (no in/out internet access) with backup and encryption enabled.
-MySQL (RDS Aurora) is running on R5 large instances.
-Redis (ElastiCache cluster) is running on M5 large instances.
-
-### S3
-Private encrypted bucket for editor and easy image files.
-
-### VPC
-Including three subnets:
+3. Change the version of the application in `aws/ecs/main.tf`. Refer to https://ckeditor.com/docs/cs/latest/onpremises/cs-onpremises/changelog.html for the list of currently available versions. You can also change the AWS region in which the resources will be created.
```

Comment:
I would prefer to set the version by default to the `latest` and mention a step to change it manually for production deployment. You can also put the on-premises version as a variable used in the `aws/ecs/main.tf` instead of hardcoding it in the configuration directly